content/hi/docs/contribute/_index.md

```diff
@@ -0,0 +1,178 @@
+
+{{< mermaid >}}
+flowchart LR
+    subgraph second[पहला योगदान]
+    direction TB
+    S[ ] -.-
+    G[दूसरे K8s मेम्बर्स के <br> prs की समीक्षा करे] -->
+    A[पहली अच्छी समस्या के लिए K8s/website<br>की समस्या सूची पर जाए ] --> B[pr ओपन करे!!]
+    end
+    subgraph first[सूचित तैयारी ]
+    direction TB
+       T[ ] -.-
+       D[योगदान अवलोकन को पढे] -->E[K8s content<br>and style guide को पढे]
+       E --> F[Hugo page<br>content types<br>और shortcodes के बारे मे जाने]
+    end
+    
+
+    first ----> second
+     
+
+classDef grey fill:#dddddd,stroke:#ffffff,stroke-width:px,color:#000000, font-size:15px;
+classDef white fill:#ffffff,stroke:#000,stroke-width:px,color:#000,font-weight:bold
+classDef spacewhite fill:#ffffff,stroke:#fff,stroke-width:0px,color:#000
+class A,B,D,E,F,G grey
+class S,T spacewhite
+class first,second white
+{{</ mermaid >}}
+**आकृति - आपके पहले योगदान की तैयारी**
+
+- योगदान करने के विभिन्न तरीकों को जानने के लिए  [योगदानकर्ता अवलोकन](/docs/contribute/new-content/overview/) को पढे । 
+- अच्छे प्रवेश बिन्दु के लिए  [`कुबेरनेटेस /वेबसाईट ` मुद्दा सूची ](https://github.com/kubernetes/website/issues/)
+  को जाँचे । 
+- [Github का प्रयोग करते हुए पुल रीक्वेस्ट खोले  ](/docs/contribute/new-content/open-a-pr/#changes-using-github) पुराने प्रलेखन मे और मुद्दा को भरने की प्रक्रिया के बारे मे ज्यादा अच्छे से जाने । 
+- भाषा और ठीक तरीके से पुल रीक्वेस्ट के लिए दूसरे कुबेरनेटस समुदाय के सदस्यों के [पुल रीक्वेस्ट की समीक्षा करे ](/docs/contribute/review/reviewing-prs/) । 
+- कुबेरनेटस [प्रकरण ](/docs/contribute/style/content-guide/) और  
+  [स्टाइल मार्गदर्शक](/docs/contribute/style/style-guide/) को पढे ताकि आप सूचित टिप्पणी दे सके। 
+- [पेज प्रलेखन टाइप ](/docs/contribute/style/page-content-types/)
+  और  [हुगो शॉर्टकोडेस ](/docs/contribute/style/hugo-shortcodes/) के बारे मे जाने। 
+
+## अगला कदम 
+
+-  कोड संग्रह मे [लोकल क्लोन से काम करने ](/docs/contribute/new-content/open-a-pr/#fork-the-repo)
+ के बारे मे सीखे। 
+- [रिलीज मे फीचर्स को](/docs/contribute/new-content/new-features/) आलेख करे। 
+- [सिग डॉक्स](/docs/contribute/participate/) मे भाग ले  और
+  [सदस्य या समीक्षक](/docs/contribute/participate/roles-and-responsibilities/) बने। 
+                       
+- [स्थानीयकरण](/docs/contribute/localization/) शुरू करे या उसमे सहायता करे। 
+
+## सिग डॉक्स मे शामिल हो 
+
+[सिग डॉक्स](/docs/contribute/participate/) योगदानकर्ताओ का एक समूह है जो कुबेरनेटेस प्रलेखन और वेबसाईट की देख रेख और उसे प्रकाशित करता है। सिग डॉक्स मे शामिल होना कुबेरनेटेस परियोजना मे बाद प्रभाव डालने का  कुबेरनेटेस योगदानकर्ताओ (महत्ववावपूर्ण लेख या उससे अन्यथा) के लिए बेहतरीन तरीका है । 
+
+सिग डॉक्स भिन्न प्रकार से संवाद करते है :
+
+- [#sig-docs को जॉइन करे  कुबेरनेटेस स्लैक चैनल मे](https://slack.k8s.io/) । खुद का परिचय देना सुनिश्चित करे। 
+- [kubernetes-sig-docs` मैलिंग लिस्ट मे शामिल हो ](https://groups.google.com/forum/#!forum/kubernetes-sig-docs),
```

Comment:
The word `mailing` should be written as `मेलिंग` instead of  `मैलिंग`. The spelling is wrong. Apparently, it could also be written as `डाकसूची` if we are going for the proper Hindi translation.

Reply:
We generally prefer easily understandable by common people over pure Hindi translation so मेलिंग should work. Thoughts?

Reply:
Definitely. But the change has to be made because of the spelling I guess.

Reply:
Thanks for tor the suggestion @jayesh-srivastava. I changed मैलिंग to मेलिंग .